Stacks_Using_Linked_Lists/Stacks.c

```diff
@@ -89,49 +90,49 @@ void main()
         printf("\nWhich operation to perform 1.PUSH 2.POP 3.Check empty or not 4.Print top element 5.Display stack 6.Size");	 /* Menu Driven Programming */
         printf("\nEnter choice");
         scanf("%d",&c);
-        if(c == 1)
+        if (c == 1)
         {
             printf("\nEnter value to push ");
             scanf("%d",&value);
             PUSH(s,value,n);
 	    break;
         }
-        else if(c == 2)
+        else if (c == 2)
         {
  	    POP(s);
 	    break;
         }
-        else if(c == 3)
+        else if (c == 3)
         {
-	    if(!isEmpty(s))
+	    if (!isEmpty(s))
 	        printf("\nStack is not empty");
             else
-		printf("\nStack is empty");
+	        printf("\nStack is empty");
 	    break;
-	}
-	else if(c == 4)
-	{
+        }
+	else if (c == 4)
+        {
             printf("\nThe topmost element is ");
 	    peek(s);
 	    break;
-	}
-	else if(c == 5)
-	{
+        }
+	else if (c == 5)
+        {
 	    PrintStack(s);
 	    break;
-	}
-	else if(c == 6)
-	{
+        }
+	else if (c == 6)
+        {
 	    size(s);
 	    break;
-	}
+        }
 	else
-	{
+        {
             break;
-	}
+        }
         printf("\nDo u want to continue(1/0)");
         scanf("%d",&k);
-    }while(k != 0);
+    } while (k != 0);
```

Comment:
I can still see some lines indented with a tab in this if else block. Please fix.

Reply:
@SakshiShreya I have made changes.

Reply:
Not fixed yet. These are some lines: 98, 102, 103 ...